internal/pass/server.go

```diff
@@ -12,8 +20,27 @@ type Server struct {
 	addr   string
 }
 
-func NewServer(addr string) *Server {
-	pairingApp := pairing.NewPairingApp()
+func NewServer(cfg config.PassConfig) *Server {
+	var awsEndpoint *string
+	if cfg.AWSEndpoint != "" {
+		awsEndpoint = aws.String(cfg.AWSEndpoint)
+	}
+	sess, err := session.NewSession(&aws.Config{
```

Comment:
Return error instead of panic. Especially, as this is not a programming error, but a problem with AWS configuration.